requirements/edx/base.txt

```diff
@@ -16,7 +16,7 @@ dealer==2.0.4
 defusedxml==0.4.1
 django-babel-underscore==0.5.1
 markey==0.8  # From django-babel-underscore
-django-celery==3.1.16
+django-celery==3.2.1
```

Comment:
So this one is going to be super fun. The tables djcelery uses are already present on devstack/prod, but they aren't *quite* right, some things have changed since they were initially made and they break spectacularly if used as-is.

Luckily for us, the project includes migrations. Unluckily for us, they're set up very poorly. Under version `3.1.16`, django claims that djcelery has no migrations at all. This has been rectified in the latest version.

Locally I was able to fix things by:
- installing the new version
- `./manage.py lms --settings=devstack migrate djcelery --fake` - Hook the existing tables up to what django migrations think they are. They don't quite line up, but the names are right so the following can succeed.
- `./manage.py lms --settings=devstack migrate djcelery zero` - Drop all of the pre-existing tables of unknown origin
- `./manage.py lms --settings=devstack migrate djcelery` - django migrations and db tables actually match, hurray